app/models/additional_code.rb

```diff
@@ -4,6 +4,8 @@ class AdditionalCode
   include ApiEntity
   include HasGoodsNomenclature
 
+  RESIDUAL_CODES = %w[49 98 99].freeze
```

Comment:
We ought to be able to do this:

```
enum :additional_code, {
   residual: %w[49 98 99]
}
```

Reply:
You may need to add the additional code field, here: https://github.com/trade-tariff/trade-tariff-backend/blob/main/app/serializers/api/v2/additional_code_serializer.rb